iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TerminalRef.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+import java.util.Optional;
+
+import com.powsybl.commons.PowsyblException;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public final class TerminalRef {
+
+    public enum Side {
```

Comment:
Probably to avoid adding dependency to other Side definition that would not make sense. Maye there should probably be a couple of generic enum for sides used everywhere ? Like TwoSide / ThreeSide, but I don't think we have to fix that here.